pkgs/development/tools/haskell/dconf2nix/default.nix

```
@@ -0,0 +1,21 @@
+{ mkDerivation, base, containers, optparse-applicative, parsec
+, stdenv, text
+}:
+mkDerivation rec {
+  pname   = "dconf2nix";
+  version = "0.0.5";
+  src = builtins.fetchTarball {
+    url    = "https://github.com/gvolpe/dconf2nix/archive/v${version}.tar.gz";
+    sha256 = "0immbx4bgfq3xmbbrpw441nx0sdpm4cp64s7qbvcbvllp4gbivpg";
+  };
+  isLibrary    = true;
+  isExecutable = true;
+  libraryHaskellDepends = [
+    base containers optparse-applicative parsec text
+  ];
+  executableHaskellDepends = [ base ];
+  homepage = "https://github.com/gvolpe/dconf2nix";
+  description = "Convert dconf files to Nix, as expected by Home Manager";
+  license     = stdenv.lib.licenses.asl20;
+  maintainers = with stdenv.lib.maintainers; [ gvolpe ];
```

personal preference, since lib is so common
```suggestion
  license     = lib.licenses.asl20;
  maintainers = with lib.maintainers; [ gvolpe ];
```